src/systemctl/systemctl-logind.c

```diff
@@ -109,6 +109,12 @@ int logind_reboot(enum action a) {
         }
         if (r >= 0)
                 return 0;
+        if (getuid() == 0 && sd_bus_error_has_name(&error, SD_BUS_ERROR_INTERACTIVE_AUTHORIZATION_REQUIRED))
+                return log_error_errno(r,
+                                       "The current polkit policy does not allow root to ignore inhibitors in order to %s.\n"
```

Comment:
this is minimally misleading: this does require the operation, but asks for reauth. maybe be accurate here and so so, i.e. just insert "without authentication"